_data/sites.json

```diff
@@ -15657,6 +15657,17 @@
         ]
     },
 
+    {
+        "name": "Strawpoll",
+        "url": "https://strawpoll.com/account/settings/",
+        "difficulty": "easy",
+        "notes": "All of your polls will be eliminated",
+        "notes_it": "Tutti i tuoi sondaggi (poll) saranno eliminati permanentemente",
```

Comment:
I think these notes add a good detail, but mention no steps for deletion, is it very clear how-to once you're on the settings page?

Reply:
Yes, it's very clear as you see in this screenshot I took
![image](https://github.com/jdm-contrib/jdm/assets/88104208/1d203a55-d5dd-4e0e-a943-324fa39b8082)

Reply:
Thanks!